test/esm.test.ts

```
@@ -80,6 +81,35 @@ describe("genExport", () => {
   }
 });
 
+const genExportDefaultTests = [
+  { expression: "myFunction", code: 'export default myFunction;' },
+  { expression: undefined, code: 'export default undefined;' },
+  { expression: 42, code: 'export default 42;' },
+  { expression: null, code: 'export default null;' },
```

ESLint

[error] Use `undefined` instead of `null`.

(unicorn/no-null)